memory/README.md

```diff
@@ -0,0 +1,48 @@
+## Region-based Heterogeneous Memory Management
+
+With the recent explosion of deep learning, entire industries substantial concerns around how to build
```

Comment:
This paragraph reads like a scientific paper that tries to advertise the importance of the research work. :D  I guess we don't need it.

Indeed, I think we need a Wikipedia link https://en.wikipedia.org/wiki/Region-based_memory_management to explain the concept *Region-based memory management*.